quipucords/scanner/network/inspect.py

```diff
@@ -104,6 +105,19 @@ def run(self):
         reachable. Collects the associated facts for the scanned systems
         """
         # pylint: disable=too-many-return-statements, too-many-locals
+        # Make sure job is not cancelled or paused
+        if manager_interrupt.value == ScanJob.JOB_TERMINATE_CANCEL:
+            manager_interrupt.value = ScanJob.JOB_TERMINATE_ACK
+            error_message = 'Scan canceled'
+            manager_interrupt.value = ScanJob.JOB_TERMINATE_ACK
```

Comment:
Resurrecting a five yr old PR :laughing::

@kholdaway do you know why `manager_interrupt.value` must be set twice

This code was later deduplicated in #1790 but carried this pattern :thinking: